docs/sdks/server-sdks/ruby/initialization.mdx

```diff
@@ -30,6 +30,21 @@ After initialization, you can get an instance of the client using Ruby's singlet
 client = EppoClient::Client.instance
 ```
 
+## Wait for initialization to complete
+
+Eppo SDK initialization is non-blocking. This means your application can continue running while the SDK initializes in the background. However, if you call assignment methods before initialization completes, you will get default values. It is therefore recommended to wait for initialization to finish before serving assignments.
+
+Ruby SDK 3.6.0 adds a new `wait_for_initialization(timeout=1)` method that waits for initialization to complete (or timeout reached):
+```ruby
+client.wait_for_initialization(0.5)
```

Comment:
does it throw an exception on timeout or fall through?

Reply:
It falls through